### DISCUSSION TITLE
Color customization [L] and [G] chats

### DISCUSSION THREAD

**Discussion by MiXeR54** [EssentialsX/Essentials#3823] [Q&A] [answered]
Please add field in config.
Everytime i change it in ess.jar/messages.properties and it's really anoying edit that file every .jar update

**Comment by mdcfe** [comment, marked as answer]
You shouldn't be editing the EssentialsX jar at all. You can edit messages by creating a blank `messages_AA.properties` file (where AA is the language code for your current language) under `plugins/Essentials/`, then adding the messages you want to override. These will persist across EssentialsX updates.

See the [Locale](https://essentialsx.net/wiki/Locale.html#customising-messages) page on the wiki for more information.
